src/lib/arrays.js

```diff
@@ -0,0 +1,119 @@
+const getNthElement = (index, array) => {
+  return array[index % array.length];
+};
+
+const arrayToCSVString = array => {
+  return array.join(',');
+};
+
+const csvStringToArray = string => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  return array.push(element);
+};
+
+const addToArray2 = (element, array) => {
+  return array.concat(element);
+};
+
+const removeNthElement = (index, array) => {
+  if (!index) {
+    return array.shift();
+  }
+  return array.splice(index, index - 1);
+};
+
+const numbersToStrings = numbers => {
+  return numbers.map(String);
+};
+
+const uppercaseWordsInArray = strings => {
+  return strings.map(words => words.toUpperCase());
+};
+
+const reverseWordsInArray = strings => {
+  strings.map((element, index) => {
+    element = element.split('');
+    element = element.reverse();
+    element = element.join('');
+    strings[index] = element;
+  });
+  return strings;
+  /*
+  const newArray = []
+  strings.forEach(element => {
+    element = element.split('');
+    element = element.reverse();
+    element = element.join('');
+    newArray.push(element);
+  });
+  return newArray;
+  */
+};
+
+function onlyEven(numbers) {
+  const evenNumbers = numbers.filter(element => {
+    if (element % 2 === 0) {
+      return element;
+    }
+  });
+  console.log(evenNumbers);
```

Comment:
Debugging type `console` statements like this should always be removed from the code before submitting it for review. See this didnt turn up in the terminal when running the tests, this must be because this code is not being hit and therefore not tested